launcher/launcher/main.go

```diff
@@ -155,6 +156,14 @@ func startLauncher(ctx context.Context, launchSpec spec.LaunchSpec, serialConsol
 	}
 	defer containerdClient.Close()
 
+	if launchSpec.Experiments.EnableGpuDriverInstallation && launchSpec.InstallGpuDriver {
```

Comment:
Imagine a scenario when experiment is off but `InstallGpuDriver` is on, do we want to continue the launcher execution, or log the error and then exit early? I'll leave it to you to make the call.

Reply:
It would good to exit early and instead of waiting for the workload to fail.